vignettes/howtogenefilter.Rmd

```diff
@@ -0,0 +1,190 @@
+---
+title: "Using the genefilter function to filter genes from a microarray dataset"
+author:
+- name: Robert, Gentleman
+  affiliation: Vignette Author's Institution
+- name: Khadijah Amusat
+  affiliation: Vignette translation from Sweave to R Markdown / HTML
+date: March 09, 2023
+output:
+  BiocStyle: BiocStyle:: html_documents:
+    number_sections: true
+    toc: true
+    toc_depth: 4
+package: genefilter
+vignette: >
+  %\VignetteIndexEntry{Using the genefilter function to filter genes from a microarray}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+# Introduction {#introduction .unnumbered}
+
+The *genefilter* package can be used to filter (select) genes from a
+microarray dataset according to a variety of different filtering
+mechanisms. Here, we will consider the example dataset in the
+`sample.ExpressionSet` example from the *Biobase* package. This
```

Comment:
Since `Biobase` is a package, add the canonical URL using the code in https://bioconductor.org/packages/release/bioc/vignettes/BiocStyle/inst/doc/AuthoringRmdVignettes.html#4_Style_macros.